### ISSUE TITLE
media_repository worker needs "media" instead of "client"

### ISSUE THREAD

**Issue by mnowiasz** [matrix-org/synapse#6518]
### Steps to reproduce

I tried to setup workers using the documentaion (docs/workers.md) but I was at first unable to get synapse.app.media_repository working, using the worker resulted in resource not found/404. Finally after looking through the python code I found out that - contrary to the documentation - the resource needs to be named media instead of client. After that the worker worked just fine.

<!--
Describe how what happens differs from what you expected.

If you can identify any relevant log snippets from _homeserver.log_, please include
those (please be careful to remove any personal or private data). Please surround them with
``` (three backticks, on a line on their own), so that they are formatted legibly.
-->

### Version information

<!-- IMPORTANT: please answer the following questions, to help us narrow down the problem -->

<!-- Was this issue identified on matrix.org or another homeserver? -->
- **Homeserver**: matrix.netzgemeinde.eu

If not matrix.org:

<!--
 What version of Synapse is running?

You can find the Synapse version with this command:

$ curl http://localhost:8008/_synapse/admin/v1/server_version

(You may need to replace `localhost:8008` if Synapse is not configured to
listen on that port.)
-->
- **Version**: 1.6.1

- **Install method**: pip/virtualenv
<!-- examples: package manager/git clone/pip  -->

- **Platform**: Linux (Gentoo, VM)
<!--
Tell us about the environment in which your homeserver is operating
distro, hardware, if it's running in a vm/container, etc.
-->

**Comment by richvdh** [comment]
could you be clearer exactly what needs changing? A PR to fix it would be much appreciated.

**Comment by mnowiasz** [comment]
Well, just correct the documentation (docs/workers.md) to reflect the change. I can submit a PR

**Comment by mnowiasz** [comment]
Done (6519), I hope I did everything OK, I'm not *that* fluent in creating PRs :-)

**Comment by richvdh** [comment]
fixed by #6519
